components/notifier/tests/strategies/test_notification_strategy.py

```diff
@@ -159,24 +167,26 @@ def test_multiple_reports_with_same_destination(self):
         """Test that the correct metrics are notified when multiple reports notify the same destination."""
         red_metric2 = self.red_metric.copy()
         red_metric2["name"] = "metric2"
-        subject2 = {"metrics": {"metric1": red_metric2}, "type": "software"}
```

Comment:
Was it incorrect that we used `metric1` in `subject2` before?

Reply:
No, the two metrics had two different uuids (`metric_uuid` and `metric1`), so that was not incorrect, just confusing.